web_src/js/features/comp/Cropper.ts

```diff
@@ -0,0 +1,61 @@
+import Cropper from 'cropperjs';
+import {showElem} from '../../utils/dom.ts';
+
+export function initCompCropper() {
+  const cropperContainer = document.querySelector('#cropper-panel');
+  if (!cropperContainer) {
+    return;
+  }
+
+  let filename;
+  let cropper;
```

Comment:
This will break if there are multiple `#cropper-panel` on the same page. Maybe attach the cropper instance to the `input` element instead, e.g. `input._cropper` and access it with that.